sentry_sdk/profiler.py

```diff
@@ -291,11 +306,18 @@ def slice_profile(self, start_ns, stop_ns):
                                 "line": frame[2],
                             }
                         )
-                    sample["frames"].append(frames[frame])
+                    current_stack.append(frames[frame])
+
+                try:
+                    stack_idx = stacks.index(current_stack)
```

Comment:
I have no clue what the sizes of these are, but why not do the same optimization as with `frames` and `frames_list`, so have a dict `stacks_dict` and a list `stacks_list` instead of walking the current `stacks` list with `index`?

Reply:
oh ok @Zylphrex said the same, did not refresh